src/sql/parts/notebook/notebook.contribution.ts

```diff
@@ -10,13 +10,19 @@ import { SyncActionDescriptor } from 'vs/platform/actions/common/actions';
 import { Action } from 'vs/base/common/actions';
 import { IEditorService } from 'vs/workbench/services/editor/common/editorService';
 import { TPromise } from 'vs/base/common/winjs.base';
-import * as nls from 'vs/nls';
 import { Schemas } from 'vs/base/common/network';
 
 import { NotebookInput, NotebookInputModel } from 'sql/parts/notebook/notebookInput';
 import { NotebookEditor } from 'sql/parts/notebook/notebookEditor';
 import URI from 'vs/base/common/uri';
 
+import { localize } from 'vs/nls';
```

Comment:
Can you move vs references up to where all the other VS references are, and group the sql references together too? This is a minor code pattern we go for since it makes it easier to spot what we're importing from the 2 areas.